docs/test-runner.md

```diff
@@ -0,0 +1,11 @@
+# Test runner
+
+Se escoge como test runner a Cargo, como no podía ser de otra manera.
```

Comment:
Aquí exactamente igual. No se trata de que escojáis lo más fácil para el lenguaje que estéis usando, sino que aprendáis una metodología que os sirva para escoger herramientas sea cual sea el lenguaje.